src/lib/zkapps_examples/empty_update/zKApps_empty_update.ml

```diff
@@ -0,0 +1,234 @@
+open Pickles_types.Hlist
+open Snark_params.Tick
+open Currency
+open Signature_lib
+open Mina_base
+
+module Party_under_construction = struct
+  type t = { public_key : Public_key.Compressed.t; token_id : Token_id.t }
+
+  let create ~public_key ?(token_id = Token_id.default) () =
+    { public_key; token_id }
+
+  let to_party (t : t) : Party.Predicated.t =
+    { body =
```

Comment:
nit: Use `Party.Body.dummy`?

Reply:
I've deliberately avoided doing that here, with the intention of replacing those dummies with this API and using it for everything.